### PR TITLE
GL-1424 java memory for cram to unmapped bams

### PR DESCRIPTION
**Pull request by jessicaway**

### CODE REVIEW

File: pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl [hunk 1]
```diff
@@ -104,8 +104,10 @@ task RevertSam {
     Int memory = 3
```
Comment (gbggrant) [inline pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl:104]: Maybe name this 'memory_in_GiB or something like that?  I could see someone passing 3000.
And perhaps we should standardize on mb?  (not sure if this is already an use like this)

**Review by gbggrant — APPROVED** [review]

File: pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl (hunk 1 above)
Comment (jessicaway) [reply pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl:104]: I can rename to clarify. 
I see both mb and gb for memory specification in warp. I agree we should standardize. Which one is preferred?

Comment (gbggrant) [reply pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl:104]: I think mb, that we can stick with ints?

Comment (jessicaway) [reply pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl:104]: Sure, that sounds good. I will just confirm with Hornet and Hops that switching it here is not going to cause issues for them

**Review by nikellepetrillo — APPROVED** [review]

**Pull request merged**